CHANGELOG.md

```diff
@@ -12,20 +12,24 @@ realm.syncSession?.addProgressNotification(
   (estimate) => console.log(`progress: ${estimate}/1.0`)
 );
 ```
+* It is no longer an error to set a base url for an App with a trailing slash - for example, `https://services.cloud.mongodb.com/` instead of `https://services.cloud.mongodb.com` - before this change that would result in a 404 error from the server. ([realm/realm-core#7791](https://github.com/realm/realm-core/pull/7791))
+* Performance has been improved for range queries on integers and timestamps when using the `BETWEEN` operator. ([realm/realm-core#7785](https://github.com/realm/realm-core/pull/7785))
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* None
+* Opening an Flexible Sync Realm asynchronously may not wait to download all data. ([realm/realm-core#7720](https://github.com/realm/realm-core/issues/7720), since v10.12.0)
+* Clearing a list of `mixed` in an upgraded file would lead to an assertion failing. ([realm/realm-core#7771](https://github.com/realm/realm-core/issues/7771), since 12.7.0-rc.0)
+* `Realm.writeCopyTo()` on an encrypted Realm without explicitly specifying a new encryption key would only work if the old key happened to be a string without null characters. ([realm/realm-core#7842](https://github.com/realm/realm-core/issues/7842), since v12.10.0)
+* Assigning a mixed property with a list should replace the value by clearing an existing list first. ([realm/realm-core#7809](https://github.com/realm/realm-core/issues/7809), since v12.9.0)
```

Comment:
```suggestion
* A `mixed` property with a collection could sometimes end up with a combination of values assigned by different clients. ([realm/realm-core#7809](https://github.com/realm/realm-core/issues/7809), since v12.9.0)
```